src/config/config.c

```diff
@@ -153,51 +146,61 @@ static int read_main_elements(const OS_XML *xml, int modules,
 #else
             mwarn("%s configuration is only set in the manager.", node[i]->element);
 #endif
-        } else if (strcmp(node[i]->element, osgcp) == 0) {
-            if ((modules & CWMODULE) && (Read_GCP(xml, node[i], d1) < 0)) {
+        } else if (chld_node && (strcmp(node[i]->element, task_manager) == 0)) {
+#if !defined(WIN32) && !defined(CLIENT)
+			if ((modules & CWMODULE) && (Read_TaskManager(xml, node[i], d1) < 0)) {
+				goto fail;
+			}
+#else
+			mwarn("%s configuration is only set in the manager.", node[i]->element);
```

Comment:
Remove this. It is not necessary the `ifdef`.

Reply:
Done